assets/js/app/Instance.js

```diff
@@ -8,8 +8,8 @@
 
   LocusZoom.Instance Class
 
-  An instance is an independent LocusZoom object. Many instances can exist simultaneously
-  on a single page, each having its own data caching, configuration, and state.
+  An Instance is an independent LocusZoom object. Many such LocusZoom objects can exist simultaneously
```

Comment:
I like "Plot" better than "Viewer" for purely subjective reasons but that's still not perfect (it seems sort of interchangeable with Panel, or put another way: it may not be obvious by names alone that a Plot is a thing that contains Panels which in turn contains Data Layers). It's better than Instance, though.